express/src/routes/sign-in.ts

```diff
@@ -50,7 +51,20 @@ router.get("/no-account", (req, res) => {
     res.render("no-account-found.njk");
 });
 
-router.get("/forgot-password", forgotPasswordForm);
-router.get("/check-email-password-reset", checkEmailPasswordReset);
+router.get(
+    "/forgot-password",
+    emailIsPresentInSession("sign-in.njk", {errorMessages: {emailAddress: "Enter your email address"}}),
```

Comment:
We'll need to fix the repetition of error messages but that's a story for another time